torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -418,6 +418,30 @@ RegisterOperators reg(
            push(stack, a.sizes());
            return 0;
          }),
+     Operator(
+         "aten:sum(Tensor self) -> int",
+         [](Stack& stack) {
+         at::Tensor x;
+         pop(stack, x);
+         int sum;
+         for(int i = 0; i < sizeof(x)/sizeof(x[0]); i++){
```

Comment:
These implementations don't match Python's, try running something like
```python
>>> sum(torch.ones(2, 2))
tensor([2., 2.])
```

Similar to your `all` PR we also already have `sum()`, try grepping the codebase for `aten::sum` to see it

Reply:
Plus, it would be great to implement `sum` for lists of integers, or lists of lists!